metrics/query.global/global.go

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 Block, Inc.
+
+package queryglobal
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/cashapp/blip"
+	"github.com/cashapp/blip/metrics/util"
+)
+
+const (
+	DOMAIN = "query.global"
+
+	OPT_RESPONSE_TIME_PERCENTILES = "response-time-percentiles"
+	OPT_RESPONSE_TIME_OPTIONAL    = "response-time-optional"
+	OPT_TRUNCATE_TABLE            = "truncate-table"
+	DEFAULT_PERCENTILE_OPTION     = "999"
```

Comment:
@joycse06 and I talked about this, and seeing it in action again, "999" might have been a bad idea on my part. It's implicit for p999, but maybe _too_ implicit. When writing about such things, no one would write just "999"; it has to be "p999" to convey the fact that it's a percentile. The same is true for "99" and "99.9" but those are a little more clear since percentiles must range `[0, 100]`, which is why "999" is odd since it's out of range. Perhaps we make this a little more strict (i.e. the sqlutil code you factored out) and support: "0" through "100" (float) and `^[pP]\d+/$` ?

Reply:
Thought about this more and, if you could please, let's change this to "0" through "100" (float) and `/^[pP]\d+$/`.  That's more consistent with established norms (e.g. "p99" is a norm, and "99" is at least within a logical range: `[0, 100]`.)

Reply:
yes, I can refactor